hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSpark.java

```diff
@@ -107,6 +125,51 @@ public static synchronized boolean isRunningOnSpark() {
         return !RegionServerLifecycleObserver.isHbaseJVM;
     }
 
+    private static UserGroupInformation applyCreds(Credentials credentials) throws IOException {
+        LOG.info("credentials:" + credentials);
+
+        logCredInformation(credentials);
+
+        UserGroupInformation ugi = UserGroupInformation.getCurrentUser();
+        ugi.addCredentials(credentials);
+        // specify that this is a proxy user
+        ugi.setAuthenticationMethod(UserGroupInformation.AuthenticationMethod.PROXY);
+        System.out.println("Set auth to token");
+
+        LOG.info("Applied credentials");
+
+        return ugi;
+    }
+
```

Comment:
I think this would fail a security audit.

Reply:
Why?

Reply:
I think the comment is regarding writing the credential to log ("credentials:" + credentials). I do not know if the credential as is written to log will be interpreted and used. But we could log it under debug mode. BTW, we do see hbase logging certain kerberos access info in its log too. 
17/07/24 14:44:49 INFO security.AMCredentialRenewer: Attempting to login to KDC using principal: hbase/[HOST_NAME]@S[REALM_NAME]

Reply:
New change looks good.